image/armbian/files/hub-ctrl

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+while getopts "b:d:P:p:" o; do
+	case "${o}" in
+	p)
+		p=${OPTARG}
+		;;
+	*)
+		# ignore
+		;;
+	esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${p}" ]; then
+	exit 1
+fi
+
+if [ "${p}" == "0" ]; then
+	# Avoid error spam from driver
+	rmmod r8152
```

Comment:
I'm likely missing something here - What happens if we disable the LAN port and then re-enable it? Wouldn't we need a corresponding insmod or something similar, or is the module loaded automatically somehow?

Reply:
It will act as if the USB network adapter was just plugged in and load the appropriate kernel driver